packages/flutter_driver/test/src/real_tests/flutter_driver_test.dart

```diff
@@ -1055,6 +1090,15 @@ vms.Response? makeFakeResponse(
   });
 }
 
+void Function(vms.Isolate) changeIsolateEventAfter(int gets, vms.Event nextEvent) {
+  return (vms.Isolate i) {
+    gets--;
```

Comment:
nit: style guide for Flutter repo says to use `-=`.

Reply:
Sorry, must have missed that. Will fix (in a couple of days if that's not a problem)!

Reply:
No worries